packages/ERTP/src/issuer.js

```diff
@@ -20,7 +27,7 @@ import './types';
 function makeIssuerKit(allegedName, amountMathKind = MathKind.NAT) {
   assert.typeof(allegedName, 'string');
 
-  const brand = Remotable(`Alleged: ${allegedName} brand`, undefined, {
+  const brand = Remotable(makeBrandInterface(), undefined, {
```

Comment:
Oops. All of these are missing the `allegedName` argument!

Reply:
Yup, fixed!